src/gui/qgsattributeform.cpp

```diff
@@ -1258,6 +1258,27 @@ bool QgsAttributeForm::currentFormValidConstraints( QStringList &invalidFields,
   return valid;
 }
 
+bool QgsAttributeForm::currentFormValidHardConstraints( QStringList &invalidFields, QStringList &descriptions ) const
+{
+  bool valid( true );
```

Comment:
We no longer like good old `=`?

Reply:
Not my code: it's a copy and paste from the pre-existing `bool QgsAttributeForm::currentFormValidConstraints` I would have used brace initializers.